cetlib/test/filepath_maker_test.cc

```diff
@@ -7,15 +7,15 @@
 #define BOOST_TEST_MODULE (filepath_maker test)
 #include "boost/test/unit_test.hpp"
 
-#include "boost/filesystem.hpp"
+#include <filesystem>
 #include "cetlib/filepath_maker.h"
 #include "cetlib/filesystem.h"
 #include "cetlib/getenv.h"
 #include "cetlib_except/exception.h"
 
 #include <string>
 
-namespace bfs = boost::filesystem;
+namespace bfs = std::filesystem;
```

Comment:
Retaining `bfs` as a name seems incongruous: could we replace it with `fs`, maybe?